docs/en_US/README.md

```diff
@@ -44,7 +44,7 @@ Opera  | [Download](https://addons.opera.com/en/extensions/details/tampermonkey-
 For other browsers you can download Tampermonkey on [tampermonkey.net](https://www.tampermonkey.net/).
 
 ::: warning
-Please note that we do not officially support older browsers, mobile browsers and Microsoft Edge or Internet Explorer. Support for these browsers is therefore neither guaranteed nor likely.
+Please note that we do not officially support; older browsers; mobile browsers; Microsoft Edge; or Internet Explorer.
```

Comment:
Please have a look at the large amount of `;`s they do not seem right in here 😉

Reply:
Semi colons (`;`'s) are indeed correct here where we are listing different browser variants.
I have copied through the semi colons to all instances of this warning, since the list of browsers is a complex list.
Commas would still be correct if you prefer that style.

Reply:
Yes, I would preffer commas tbh. Semicolons are kind of a "very hard break" for me, not for a list of values :)